package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "private": true,
+  "devDependencies": {
+ },
+  "scripts": {
+    "build": "node ./node_modules/gulp/bin/gulp.js",
+    "ensure-images": "node ./node_modules/gulp/bin/gulp.js ensure-images-optimized",
+    "serve": "yarn run ensure-images && node ./node_modules/polyserve/bin/polyserve -p 5101 -P h2 --proxy-path api --proxy-target http://localhost:9000/api/"
+  },
+  "engines": {
+    "node": ">=5.0.0"
```

Comment:
@TimvdLippe we are running on travis node version 6.0.0, should we update this, too?